<launch>

  <arg name="server" default="mocap.local"/>

  <node pkg="vrpn_client_ros" type="vrpn_client_node" name="vrpn_client_node" output="screen">
    <rosparam subst_value="true">
      server: $(arg server)
      port: 3883

      update_frequency: 100.0
      frame_id: world

      # Use the VRPN server's time, or the client's ROS time.
      use_server_time: false
      broadcast_tf: true

      # Must either specify refresh frequency > 0.0, or a list of trackers to create
      refresh_tracker_frequency: 1.0
      #trackers:
      #- FirstTracker
      #- SecondTracker
    </rosparam>
    <!-- <remap from="vrpn_client_node/aslquad/pose" to = "/mavros/vision_pose/pose" /> -->
  </node>

  <node pkg="asl_flight" type="vrpn2pose" name="vrpn2pose" output="screen">
		<param name="vrpn_frame" value="/vrpn_client_node/aslquad/pose" />
		<param name="publish_frequency" value="100" />
		<param name="is_stamped" value="true" />
	</node>

</launch>
